test/test_images/multicontainer/servingcontainer/servingcontainer.go

```diff
@@ -22,13 +22,18 @@ import (
 	"io"
 	"log"
 	"net/http"
+	"os"
 
 	"knative.dev/serving/test"
 )
 
+const (
+	defaultPort = "8080"
+)
+
 func handler(w http.ResponseWriter, r *http.Request) {
 	log.Println("serving container received a request.")
-	res, err := http.Get("http://127.0.0.1:8882")
+	res, err := http.Get(os.ExpandEnv("http://localhost:$FORWARD_PORT"))
```

Comment:
How does the env relate to the `getPort` below? One now seems to have defaulting while the other one has not. It's a bit hard to read.

Reply:
`getPort` is for PORT env variable, it provides a default when PORT is not specified. It's pretty common to not include PORT because it's automatically added for the user container when the user doesn't specify one. 
If someone wants to use `FORWARD_PORT`, they need to specify it explicitly. Defaults don't make much sense here, IMO. It could be some random port but the value of adding it would be low. Any suggestions?

Reply:
Hm makes sense, I agree that a random port does not make much sense. Maybe it's just a naming thing. Maybe call the `getPort` here `getServerPort`?